lib/internal/Magento/Framework/Stdlib/Test/Unit/DateTime/TimezoneTest.php

```diff
@@ -253,4 +253,18 @@ private function scopeConfigWillReturnConfiguredTimezone($configuredTimezone)
     {
         $this->scopeConfig->method('getValue')->with('', '', null)->willReturn($configuredTimezone);
     }
+
+    /**
+     * Test checks, if  method "scopeDate" set timezone value.
```

Comment:
Make a proper test name instead.

Reply:
and remove description ?

Reply:
@Nazar65 ideally yes but I'm not sure it passes current (lame) sniffs

Reply:
@orlangur yes checked and test passed.

Reply:
@orlangur done.